FFmpegConfig.sh

```diff
@@ -1,321 +1,130 @@
 #!/bin/bash
 DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
-if [ "$1" == "Win10" ]; then
-    echo "Make Win10"
-
-    if [ "$2" == "x86" ]; then
-        echo "Make Win10 x86"
-        pushd $DIR/ffmpeg
-		rm -rf Build/Windows10/x86
-        rm -rf Output/Windows10/x86
-        mkdir -p Output/Windows10/x86
-        cd Output/Windows10/x86
-        ../../../configure \
-        --toolchain=msvc \
-        --disable-programs \
-        --disable-d3d11va \
-        --disable-dxva2 \
-		--disable-encoders \
-		--disable-devices \
-		--disable-hwaccels \
-        --disable-doc \
-        --arch=x86 \
-        --enable-shared \
-        --enable-cross-compile \
-        --enable-debug \
+declare -A arch
+arch['x86']='x86'
+arch['x64']='x86_64'
+arch['ARM']='arm'
+arch['ARM64']='arm64'
+
+variant=$1
```

Comment:
Good job on clening up the script! Our whole library only supports Windows 10. So we could even go further and delete all other variants to further simplify the script...? Then also change folder name to "ffmpeg" instead of "ffmpeg-Win10".

Reply:
Love the idea.

Please note, for some reason, this breaks the sample app even if  the UWP library still builds (and works as an exportable NuGet).

I'll address the feedback but tag this PR as a draft until it's properly tested.